src/helpers/document.js

```diff
@@ -130,8 +148,19 @@ const checkForUpdates = () => {
             return response.json();
         })
         .then(data => {
-            //TODO: get the correct version number and compare which is greater. Semver compare.
-            if (!data[0].name.includes(version)) {
+            const lastVersion = data[0].name.substr(1).split('.');
```

Comment:
As we talked, since those semver comparer are doing the same as we are doing here, we decided to stay with this code instead of using a package for it.